# Fixed Genes
int32 SimulationState # 0->stop 1->run 2->pause
